examples/sample-primitives-project/build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.9.0"
+
+    // Import the Avail Plugin into the build script
+    id("org.availlang.avail-plugin") version "2.0.0.alpha20"
+}
+
+group = "org.availlang"
+version = "1.0.0.alpha01"
+description = "JVM-based primitives written in Kotlin to be accessed using " +
+    "Avail's primitive linker"
+
+repositories {
+    mavenCentral()
+}
+
+val targetJVM = 17
+val kotlinLanguage = 1.7
```

Comment:
We're not targeting Kotlin language 1.8 here?

Reply:
set it to 1.9